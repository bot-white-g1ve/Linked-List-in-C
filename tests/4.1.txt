AVX
()
{0}
1 VIEW