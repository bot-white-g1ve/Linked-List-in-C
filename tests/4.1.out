INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
